### PR TITLE
Reworked Block Stats Table View #195

### PR DESCRIPTION
**Pull request by Macdonny**
Reworked blocked stats table.

I divided the different datasets into their own tables, which can be accessed by clicking the tabs. However, the addition of the four tables is a slight deviation from the original request in the issue, so let me know if you still want the last blocked datasets combined instead.

https://github.com/user-attachments/assets/90b2961c-2ab4-477c-b755-c52e769c408a

### CODE REVIEW

**Comment by thieflord06** [comment]
Looks great! My only suggestion/question is how can this be formatted better for mobile? All the data doesn't fit and there is some horizontal swiping that has to be done to see everything.

**Comment by Macdonny** [comment]
> Looks great! My only suggestion/question is how can this be formatted better for mobile? All the data doesn't fit and there is some horizontal swiping that has to be done to see everything.

Thanks! I also added a change to make the tabs more mobile-friendly. 

Horizontal scrolling is a decent solution for mobile users, but I agree it isn't perfect. There are alternatives like displaying limited columns and then displaying the rest of the data in a click popup/hover or another page or another option would be displaying the row data vertically, something like this:

image

However, from what I see in the Ag Grid library, it might not be the best library if we choose to go with any of the alternatives. Saying all that, it'll take some considerable refactoring of this table into one of the alternatives, and we might want to wireframe it before overhauling it.

**Comment by thieflord06** [comment]
There were discussions in #design about moving from ag grid. I'll let @noahm chime in on how this is so far for this PR. With us moving semi-full-steam-ahead on a redesign, I'm very hesitant to put too many changes to make the current one work.

File: src/landing/home-stats/home-stats-table.jsx [hunk 1]
```diff
@@ -211,3 +259,16 @@ function getLabelForStatKey(key) {
...
 }
+
+/**
+ * @param {import('ag-grid-community').ICellRendererParams} params
+ * 
+ * @returns {JSX.Element | string}
+ */
+function HandleLinkRenderer(params) {
+    try{
+      return (<a href={new URL(params.data.ProfileURL).href} target="_blank">{params.data.Handle}</a>);
```
Comment (noahm) [inline src/landing/home-stats/home-stats-table.jsx:270]: do we want to link up bsky or their profile on clearsky?

**Review by noahm — APPROVED** [review]
This looks all entirely reasonable. Like @thieflord06  says I don't think were all that committed to staying with ag-grid in the medium term, but this is a fine step forward for now.

**Review by thieflord06 — APPROVED** [review]

File: src/landing/home-stats/home-stats-table.jsx (hunk 1 above)
Comment (Macdonny) [reply src/landing/home-stats/home-stats-table.jsx:270]: Yeah, linking this to their profile on ClearSky makes sense. I can handle this in another PR.

**Pull request merged**